### ISSUE TITLE
Need Trained models

### ISSUE THREAD

**Issue by NainaSaid** [markus93/NN_calibration#15]
Hi Markus,

I cant seem to download the trained models due to bandwidth issues. Do you have the trained models saved somewhere I can access? I am asking for the models and not the logits. Please help!

Thanks

**Comment by markus93** [comment]
Hey!

Most of the models are available [here ](https://owncloud.ut.ee/owncloud/s/8wjqyeM2aKfZoGp). I will add this link also to the models' folder.

Best wishes,
Markus
